pages/rdm_cycle/analysing.md

```diff
@@ -1,20 +1,39 @@
+
+Data analysis encompasses all the different data manipulation and transformations that will help scientists to discover information or generate new knowledge.
+This is the step where the actual work on the data towards the goal of a research project takes place.
+The steps of the workflow in the Analysis phase of a project will often be repeated several times to explore the data as well as to optimize the process.
+According to the different types of data (quantitative or qualitative) the methods will differ.
 
 Data analysis follows the (often automated, batch) processing in the Processing stage.
 
 ## Why is Data Analysis important?
-Data Analysis is the main reason for a data intensive project. It actually works towards achieving the goal.
+
+Since it is the stage where new knowledge and information are generated, it can be considered as central in the research process.
+With many disciplines becoming data-oriented, more and more data intensive projects will occur and will involve experts from many thematic fields.
 
 ## What should be considered for Data Analysis?
-* Provenance: it is important that the exact steps that have been taken in the data analysis are documented. This includes the exact version of the software used, as well as any parameters. Manual "manipulation" of the data may complicate this documentation process.
-* Who will be doing the analysis? Will it be a computer wizard, or a subject matter expert? In the first case, a series of tools that can be used from the unix command line may give a lot of flexibility, in the latter case a workflow system (such as Galaxy) may be better.
+
+Because of their nature, data in Life Sciences are now considered as Big Data. These characteristics of Big Data, often summarized by a growing list of "V" properties (Volume, Velocity, Variety, Veracity, Value, etc.), impact strongly the methods and technical solutions used for Data Analysis.
+
+* By its volume and velocity, data generated in Life Sciences, create a challenge that has to be addressed at several levels. At the storage level, including the transfer of the data from the data production facility to the computing facility for its analysis. It is sometimes worth to consider the cost of the transfer of massive amounts of data compared to the transfer of virtual images of machines for the analysis of the data.
+* The variety of the data poses an integration challenge that can only adressed with the help of best practices that make data interoperable and reusable.
+* The Data Analysis phase relies on the previous steps (collection, processing) that will lay the foundations for the generation of new knowledge by providing acurate and trustworthy data.
+
+## Problems to be addressed at this stage
```

Comment:
Probably the text under this section should be a part of the "Considerations" section. In the other pages we mostly add links of related topics in this section.
Maybe we can discuss it tomorrow in the editorial meeting.